tests/Schema/Keywords/MultipleOfTest.php

```diff
@@ -5,39 +5,76 @@
 namespace OpenAPIValidationTests\Schema\Keywords;
 
 use OpenAPIValidation\Schema\Exception\ValidationKeywordFailed;
-use OpenAPIValidation\Schema\Validator;
+use OpenAPIValidation\Schema\SchemaValidator;
 use OpenAPIValidationTests\Schema\SchemaValidatorTest;
 
 final class MultipleOfTest extends SchemaValidatorTest
```

Comment:
Please review this test. It fails at the moment.

Reply:
Weird, but fixed. int/int is int if it's divisible, not float. what the hell. added explicit cast to float for comparison

Reply:
As it turns out, the result of a division can be both `int` and `float`. It would be better to have `float` as a result of a division in all cases (thought PHP is not there yet).
```php
4/2 === 2; // true
4.0/2 === 2.0; //true
```

Reply:
yeah, that's the thing I did in the fix. Updated it to be more clear